### ISSUE TITLE
lint: add empty-fmt checker

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#107]
For some reasons, this kind of code exists in our own codebase:
```go
x := fmt.Sprintf("abc")
```
It uses formatted print function, but does not specify and formatting arguments, neither string itself has any formatting directives.
If there are actually missing arguments for format directives, go vet gives a warning:
```go
x := fmt.Sprintf("%d")
// Sprintf format %d reads arg #1, but call has 0 args (vet)
```

So, our task is to check formatted IO calls for arguments count.

For `fmt.Sprintf` it's a bad practice to use 1 argument.
For `fmt.Fprintf` it's a bad practice to use 2 arguments (because 1st argument is a file + formatting string itself).
And so on. There are many such functions in go, `log` package is a second lead.

This checker should be implemented using `baseExprChecker`.

**Comment by Quasilyte** [comment]
Should wait until #106 is resolved.
Or can be implemented in an old fashion. I'll port it to new framework quite easily.

**Comment by Quasilyte** [comment]
Note that user-defined functions with format strings are OK.
We should only check for very common cases which we 100% sure about.
For example, `fmt.Sprintf("abc")` is dumb and is never useful,
on the other hand `logErrorf("abc")` has unknown properties and should be left alone.
There could be no just `logError` so the format string without arguments (and directives) are fine there.
